tests/components/modbus/test_sensor.py

```diff
@@ -686,7 +686,7 @@ async def test_config_wrong_struct_sensor(
             },
             [0x00AB, 0xCDEF],
             False,
-            "112593.75",
+            "112594",
```

Comment:
Why is it needed to change this?

Reply:
Because now it return an integer which is rounded, and not a float.